### DISCUSSION TITLE
implementation of the bias term

### DISCUSSION THREAD

**Discussion by gokhalen** [neuraloperator/neuraloperator#460] [General]
In your paper "Fourier Neural Operator For Parametric Partial Differential Equations" ICLR 2021,  you have in equation (2) a linear transformation $W:\mathbb{R}^{d_v}\rightarrow \mathbb{R}^{d_v}$. However, this seems to be missing in the SpectralConv [here](https://github.com/neuraloperator/neuraloperator/blob/95b9a5997039cbde899131aefb9284561dfdef77/neuralop/layers/spectral_convolution.py#L484), in favor of a bias value which is added by broadcasting to each channel. 

I wonder why this change was made, to me the equation (2) in the paper seems more general than the current implementation.

**Comment by JeanKossaifi** [comment]
We are differentiating the Spectral Convolution from the FNO Block which is what you are referring to. The linear transformation is there, along with other options: https://github.com/neuraloperator/neuraloperator/blob/95b9a5997039cbde899131aefb9284561dfdef77/neuralop/layers/fno_block.py#L59 
I'd recommend using directly through the [FNO](https://neuraloperator.github.io/dev/modules/generated/neuralop.models.FNO.html#neuralop.models.FNO) interface though.

**Reply by gokhalen** [reply]
Thanks! That clears it up.
